### DISCUSSION TITLE
Fabric API not opening?

### DISCUSSION THREAD

**Discussion by hobbestopia** [FabricMC/fabric-api#1312] [FAQ] [answered]
Whenever I try to open Fabric API, it keeps saying that I dont have java installed but I do
How do I fix this? 

Thank you!

**Comment by Daomephsta** [comment, marked as answer]
Fabric API is a standard Fabric mod. It's not installed by running it, only fabric-loader needs to be installed that way.

**Reply by liach** [reply]
However, if you install fabric with fabric's installer, you do need to install java to run that installer.

**Reply by XfedeX** [reply]
Dshash how can you even play minecraft if you have no Java installed?!?!

**Reply by liach** [reply]
@XfedeX You use the java from the minecraft launcher. Windows cannot automatically detect that java as a way to run java programs unless the user sets it up explicitly use "open with".
